src/components/evaluate.ts

```diff
@@ -19,7 +19,9 @@ export function evaluate(str: string, context: any, currentProp: string, fallbac
   try {
     return compile(str)(_context);
   } catch (err) {
-    console.log(err, str, _context);
+    if (debug) {
+      console.log(err, str, _context);
```

Comment:
The reason I didn't put this console.log behind a debug flag is that it shouldn't _ever_ be happening if we write our `schema`s correctly. And putting it behind a debug flag will just hide the error silently.

Maybe I should make it throw instead?

Reply:
will update!

Reply:
Then do we not need the fallback value also?

Reply:
Hmm... good point. OK. If I recall, the reason I did it this way was because simply throwing the error was because throwing the error propagated up to `<ErrorBoundary>` and was unrecoverable. So even if you fixed the schema, the hot-reloading or whatever wouldn't make the Formtron usable again. :/

Really I guess what it should do is log it to the Studio logging environment. :/ For now.... yeah lets just use what you have.